addons/mail/static/src/owl/components/chatter_topbar/chatter_topbar.scss

```diff
@@ -8,15 +8,38 @@
     justify-content: space-between;
 }
 
+.o_ChatterTopbar_buttonAttachmentsCountLoader {
+    margin-left: 2px;
+}
+
 .o_ChatterTopbar_buttonCount {
-    padding-left: 0.25rem !important;
+    padding-left: 0.25rem;
 }
 
+.o_ChatterTopbar_buttonSendMessage {
+    padding: 0 16px;
+    border-left: none;
```

Comment:
If you do that "Send message" is translated of 1 px to the right because he still has a transparent border when it's not active which I think it's useless, don't you ?

Reply:
ok